src/plugins/scene3d/Scene3D.cc

```diff
@@ -210,18 +209,12 @@ namespace plugins
     /// \brief Mouse event
     public: common::MouseEvent mouseEvent;
 
-    /// \brief Mouse move distance since last event.
```

Comment:
How about we leave `Scene3D` as it is and focus on `MinimalScene`? So Edifice users can migrate to Fortress and the scene still works without any changes, but then they have the option of migrating to `MinimalScene` + other plugins.

Reply:
https://github.com/ignitionrobotics/ign-gui/pull/231/commits/0f4954cc1c4e87234de823950938375aad3875d3